### PR TITLE
restructure folders and create admin tab

### PR DESCRIPTION
**Pull request by wshep17**
This commit: 
(1) Reorganize folders for project for better visibility/organization as we scale the codebase.
(2) Add additional tab for admin user(front end only)

**Update: No need for React-Routing.**

### CODE REVIEW

**Review by telarmago — COMMENTED** [review]
Minor detail, but could we make `src/CSS` and `src/SCRIPTS` lowercase?

**Review by krschaffer — COMMENTED** [review]
I think it looks good, but I do have a couple concerns.
1. It's a little unusual to have two component libraries (bootstrap and ant design), so if we can avoid that, I think that would be a better design. If it's only possible for you to do what you want to do with bootstrap though, that's okay.
2. I don't think the chat rooms should be a tab. I would advocate for a button or some sort of pullout menu that the admins would use to access rooms. For now, we can leave it as a tab, but when we get around to implementing that, I would like to use a different component for that feature

**Comment by wshep17** [comment]
made the necessary changes you all requested. regarding the user rooms, we will discuss this as a group tomorrow on a design since this aspect isn't addressed in our wireframe and consult our client with that final idea. I will let someone else check and merge with development.

File: loan-calculator/src/index.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import 'bootstrap/dist/css/bootstrap.css';
```
Comment (telarmago) [inline loan-calculator/src/index.js]: If we're dropping bootstrap, do we still want the `npm` packages in `package.json`?

File: loan-calculator/package.json [hunk 2]
```diff
@@ -7,7 +7,9 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "antd": "^4.6.1",
+    "bootstrap": "^4.5.2",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.3.0",
```
Comment (telarmago) [inline loan-calculator/package.json]:
```suggestion
    "react": "^16.13.1",
```

File: loan-calculator/src/index.js (hunk 1 above)
Comment (wshep17) [reply loan-calculator/src/index.js]: good catch. ill remove that now.

File: loan-calculator/package.json [hunk 3]
```diff
@@ -9,7 +9,6 @@
     "antd": "^4.6.1",
     "bootstrap": "^4.5.2",
```
Comment (telarmago) [inline loan-calculator/package.json]: Did we still need this `bootstrap` package as well?

Comment (wshep17) [reply loan-calculator/package.json]: oh snap..

**Review by telarmago — APPROVED** [review]
Awesome looks good!

**Pull request merged**